notebook/static/tree/js/notebooklist.js

```diff
@@ -935,6 +951,22 @@ define([
         });
     };
 
+    NotebookList.prototype.view_selected = function() {
+        var that = this;
+        that.selected.forEach(function(item) {
+            var item_path = item.path;
+            var w = window.open(utils.url_path_join('/files', item_path));
```

Comment:
Ok, thanks for the hint. I don't think I saw any other examples in that file, but maybe I missed them, or the code is in a different file.